app/config/question-bank.js

```diff
@@ -1591,14 +1592,6 @@ const questionBank = {
             html: `For example, you will now sell directly to retailers 
                   <br/><br/>Select all that apply`
           },
-          ineligibleContent: {
```

Comment:
think we still need this ineligible, as if a user chooses the 5th option it should be notEligible (see STANDALAONE_ANSWER validation)